environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/base/SecurityAccessBase.java

```diff
@@ -14,11 +14,11 @@
         message = "Please set either only the CIDR field or both security group id fields")
 public abstract class SecurityAccessBase {
 
-    @Size(min = 1, max = 255, message = "The length of the security group ID can be minimum 1 and maximum 255 characters.")
+    @Size(min = 1, max = 4000, message = "The length of the security group ID can be minimum 1 and maximum 4000 characters.")
     @ApiModelProperty(EnvironmentModelDescription.KNOX_SECURITY_GROUP)
     private String securityGroupIdForKnox;
 
-    @Size(min = 1, max = 255, message = "The length of the security group ID can be minimum 1 and maximum 255 characters.")
+    @Size(min = 1, max = 4000, message = "The length of the security group ID can be minimum 1 and maximum 4000 characters.")
```

Comment:
I have not checked it, just asking for completeness: Does this possibly affect the size / type of the associated DB table columns as well? I mean, they would have to be adapted if their type was `VARCHAR(255)`, say.

Reply:
nice catch thanks Lajos